gammapy/estimators/points/core.py

```diff
@@ -45,58 +45,56 @@ class FluxPoints(FluxMaps):
     The `FluxPoints` object is most easily created by reading a file with
     flux points given in one of the formats documented above::
 
-        from gammapy.estimators import FluxPoints
-        filename = '$GAMMAPY_DATA/hawc_crab/HAWC19_flux_points.fits'
-        flux_points = FluxPoints.read(filename)
-        flux_points.plot()
+    >>> from gammapy.estimators import FluxPoints
+    >>> filename = '$GAMMAPY_DATA/hawc_crab/HAWC19_flux_points.fits'
+    >>> flux_points = FluxPoints.read(filename)
+    >>> flux_points.plot() #doctest: +SKIP
```

Comment:
Does this fail? Is it related to #3811 ?

Reply:
No this does not fail, I checked locally. I added the SKIP because it tests a Matplotlib axis, and I don't know a good way to test that. I tried with `<AxesSubplot:xlabel='Energy [TeV]', ylabel='dnde (1 / (cm2 s TeV))'>` , but that did not work

Reply:
I guess you could suppress the return by adding a semicolon, like `flux_points.plot();`

Reply:
No I tried that already

Reply:
OK